docs/source/models/survival_analysis.rst

```diff
@@ -0,0 +1,117 @@
+=================
+Survival Analysis
+=================
+
+.. todo::
+
+	Insert summary/introduction that explains the purpose of this section, how 
+	it relates to cause modeling, and what will be covered
+
+.. contents::
+	:local:
+
+Terms and Definitions
+---------------------
+
+.. todo::
+
+	Add definitions and descriptions to terms
+
+.. list-table:: 
+   :widths: 20 20
+   :header-rows: 1
+
+   * - Term
+     - Definition
+   * - Event
+     -
+   * - Time
+     -
+   * - Censor
+     -
+   * - Observation period
+     -
+   * - Closed cohort
+     -
+   * - Open cohort
+     -
+
+Survival Function
+-----------------
+
+.. todo::
+
+	Include disclaimer that we will first discuss closed cohorts without event 
+	recurrance until a later section
+
+	Include formula for survival function, interpretation, description, etc.
+
+	Include kaplan-meier plot 
+
+
+Hazard Function
+---------------
+
+.. todo::
+	
+	Include formula for hazard function, interpretation, description, how it 
+	relates to survival function, etc.
+
+	Include graphic of observed failure rate made up of decreasing early 
+	infant mortality hazard rate, constant random hazard rate, and increasing 
+	wear out hazard rate along with corresponding survival curves
+
+	Include an example that contrasts population time (2016) with time 
+	relative to an event (years since diagnosis).
+
+Implications for Cause Model Transition Rates
+---------------------------------------------
+
+.. todo:: 
+
+	Link to cause model transition rates data source page
+
+	Briefly discuss how incidence is used as transition probability and 
+	assumption that it is constant over the timeframe it represents
+
+	Discuss how this assumption is valid for constant hazard rates, with 
+	examples
+
+	Discuss how this assumption is less valid for the following scenarios, 

```

Comment:
This comment doesn't apply to this pull request specifically, but it's something to think about as you're writing this section:

There's some subtlety in determining what is a "valid assumption," because what is valid depends on the context in which the assumption is being used.

For example, just because the hazard function is not constant does not mean it is inappropriate to replace it with its average (i.e. the incidence rate) in a model - it depends on what we want the model to do.

If all we need the model to do is compute the incident cases in a specified time period, then replacing the nonconstant hazard function with its average over that period will still give the right answer.  On the other hand, if we need to model variability in incident cases over the time period, then we need more localized data about the hazard function than the overall average.  So the validity of an assumption depends on the goals of the model as well as the underlying process and data.

Reply:
Good point! Thank you